akka-stream-tests/src/test/scala/akka/stream/scaladsl/FlowMapAsyncSpec.scala

```diff
@@ -247,7 +247,7 @@ class FlowMapAsyncSpec extends AkkaSpec with ScalaFutures {
       }
 
       try {
-        val N = 100000
+        val N = 10000
```

Comment:
had to decrease otherwise it times out on Jenkins (takes ~5 seconds). I think with the fuzzer less iterations are enough to exercise this logic well, so I believe this is a safe modification.